src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.File.Base.Unix.cs

```diff
@@ -20,7 +21,7 @@ protected void VerifyPlatformSpecificMetadata(string filePath, TarEntry entry)
 
             if (entry is PosixTarEntry posix)
             {
-                string gname = Interop.Sys.GetGroupName(status.Gid);
+                Interop.Sys.TryGetGroupName(status.Gid, out string gname);
```

Comment:
should we assert that it succeeds?

```suggestion
                Assert.True(Interop.Sys.TryGetGroupName(status.Gid, out string? gname));
```